<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreditCalc</class>
 <widget class="QWidget" name="CreditCalc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="line_sum_credit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>151</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_srok">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>70</y>
     <width>151</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_stavka">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>110</y>
     <width>151</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="SumCredit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Сумма кредита</string>
   </property>
  </widget>
  <widget class="QLabel" name="Srok">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Срок</string>
   </property>
  </widget>
  <widget class="QLabel" name="ProzentStav">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Процентная ставка</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Annuit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>181</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Аннуитичный</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Diff">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>181</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Дифференцированный</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>150</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Ежемесячный платёж</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>180</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Переплата по кредиту</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>210</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Общая выплата</string>
   </property>
  </widget>
  <widget class="QLabel" name="mounth_pay">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>150</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="over_pay">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>180</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="final_sum">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>210</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
